server/index.js

```diff
@@ -4,24 +4,85 @@ const path = require('path');
 const logger = require('morgan');
 const http = require('http');
 const socketIO = require('socket.io');
-
+const bodyParser = require('body-parser');
+const passport = require('passport');
+const FacebookStrategy = require('passport-facebook').Strategy;
+const expressSession=require('express-session');
+const cookieParser = require('cookie-parser');
+require("dotenv").config()
 /* Init */
 const app = express();
 const server = http.createServer(app);
 const io = socketIO.listen(server);
+/* DB  */
+const users = require('./db/connection').users;
 
 /* Middleware */
-
+app.use(cookieParser());
 app.use(logger('dev'));
-
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
 const pathToStaticDir = path.resolve(__dirname, '..', 'client/public');
 app.use(express.static(pathToStaticDir));
+app.use(expressSession({ secret: 'keyboard cat', resave: true, saveUninitialized: true }));
+app.use(passport.initialize());
+app.use(passport.session());
+
+passport.use(new FacebookStrategy({
+  clientID: process.env.client_Id,
+  clientSecret: process.env.client_Secret,
+  callbackURL: "http://localhost:3000/auth/facebook/callback"
+},
+
+  (accessToken, refreshToken, profile, done) => {
+    console.log('this is the profile', profile.id);
+    users.findAll({ where: { facebookId: profile.id }
+  }).then(user => {
+    if (user.map(ind => {
```

Comment:
Could you please put this condition on 1 line? Better yet, save the value into a variable & use that in the condition.